Update prometheus-metrics-scrape-configuration.md (#119757)
Azure/azure-content: articles/azure-monitor/containers/prometheus-metrics-scrape-configuration.md

Four different configmaps can be configured to provide scrape configuration and other settings for the metrics add-on. All config-maps should be applied to `kube-system` namespace for any cluster.

> > [!NOTES]

Reviewer: ```suggestion
> [!NOTE]
```

> None of the four configmaps exist by default in the cluster when Managed Prometheus is enabled. Depending on what needs to be customized, you need to deploy any or all of these four configmaps with the same name specified, in `kube-system` namespace. AMA-Metrics pods will pick up these configmaps after you deploy them to `kube-system` namespace, and will restart in 2-3 minutes to apply the configuration settings specified in the configmap(s).
>

> > Support for Windows Kubernets in ARC are not supported. 

Reviewer: ```suggestion
> Support for Windows Kubernetes in Azure Arc isn't supported. 
```
